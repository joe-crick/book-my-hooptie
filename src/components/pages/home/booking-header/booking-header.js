import React from "react";
import styles from "./booking-header.less";
import { translate } from "react-i18next";

export const BookingHeader = ({ t: translate }) => (
  <header className={styles.header}>
    <h1 className={styles.title}>{translate("header.title")}</h1>
    <p className={styles.subtitle}>{translate("header.subtitle")}</p>
  </header>
);

export default translate()(BookingHeader);
