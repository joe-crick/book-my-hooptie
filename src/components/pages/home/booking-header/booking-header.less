@import "../../../../style-resources/mixins.less";

.header {
    display: flex;
    flex-direction: column;
    justify-content: flex-start;
}