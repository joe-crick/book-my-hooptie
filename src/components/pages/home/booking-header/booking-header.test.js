import React from "react";
import { BookingHeader } from "./booking-header";
import renderer from "react-test-renderer";
import { i18nMock } from "test/test-utils";

describe("Header", () => {
  it("should render", () => {
    const component = renderer.create(<BookingHeader t={i18nMock} />);
    let header = component.toJSON();
    expect(header).toMatchSnapshot();
  });
});
