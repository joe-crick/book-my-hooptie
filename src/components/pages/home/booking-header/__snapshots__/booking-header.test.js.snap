// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render 1`] = `
<header
  className={undefined}
>
  <h1
    className={undefined}
  >
    header.title
  </h1>
  <p
    className={undefined}
  >
    header.subtitle
  </p>
</header>
`;
